src/test/.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+    "env": {
+      "jest": true,
+      "es6": true,
+    },
+    "rules": {
+      "react/display-name": "off",
+      "react/prop-types": "off"
+    }
+  }
```

Comment:
Nitpick, but newline at the end of file. It results in weird diffs on github :p

Reply:
I have fixed the end of the file.